roles/openshift_node/handlers/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-- name: restart openshift-node
-  service: name=openshift-node state=restarted
+- name: restart node
+  service: name={{ product_type }}-node state=restarted
```

Comment:
Spoke with @detiber and sounds like the best course of action is to create a new ```service_type``` variable based upon the ```deployment_type```.

| deployment_type | product_type | service_type |
--------- | ---------------------- | --------------------- |
| origin | openshift | origin |
| origin | atomic-enterprise | origin |
| online | openshift | atomic-openshift |
| online | atomic-enterprise | atomic-openshift |
| enterprise | openshift | atomic-openshift |
| enterprise | atomic-enterprise | atomic-openshift |